icons/lootcrate.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Lootcrate</title><path d="M21.181 11.048c-.085.055-.064.17.036.201 1.885.579 2.783.66 2.783 2.081v4.011c0 .716-.06.88-.618 1.254-1.019.683-7.192 4.747-7.433 4.913-.9.62-1.36.556-2.388.339A678.009 678.009 0 0 1 1.996 21.27C-.034 20.8.018 19.206.018 18.067.018 14.122 0 10.6 0 6.865c0-1.186.057-1.222 1.104-1.88.966-.608 4.997-3.136 7.157-4.5.643-.406.961-.607 1.88-.406 1.88.41 7.75 1.65 10.91 2.339C23.476 2.947 24 3.31 24 4.744V8.38c0 .465-.008.638-.474.98-.235.174-1.568 1.14-2.345 1.688zm1.889-2.556c.083.019.164-.037.164-.112 0-1.414.013-2.495.013-3.666 0-.622-.408-1.137-1.304-1.343C17.235 2.29 10.24.866 9.897.778c-.23-.06-.375.108-.375.199 0 4.294.033 7.832.033 11.854 0 .74.475 1.525 1.503 1.765 4.165.973 7.65 1.655 12.014 2.667.084.019.166-.036.166-.112.002-.849.016-1.405.016-1.95 0-2.27.325-2.68-1.825-3.194-2.556-.612-6.171-1.378-6.458-1.44-.565-.125-.638-.267-.638-.592 0-.122-.013-2.346-.004-2.869.002-.157.007-.31.183-.397.156-.077.307-.046.772.053.52.111 5.347 1.186 7.786 1.73zm-8.897 14.735c.084.019.165-.036.165-.11.002-1.186.02-2.485.02-3.353 0-1.475.213-1.334-1.237-1.639-1.713-.36-5.155-1.146-6.939-1.552-.413-.094-.534-.424-.534-.998 0-1.726-.007-4.575-.007-6.577 0-1.834.224-1.599-.818-1.832-.657-.146-2.87-.637-3.852-.828-.167-.032-.261.085-.261.224-.004 3.94.061 8.21.061 11.994 0 1.228.51 1.815 1.536 2.051 4.173.963 7.496 1.652 11.866 2.62z"/></svg>
```

Comment:
I'm seeing some differences - looks like the icon has been stretched to fit the square viewport. Could you please amend to keep the correct aspect ratio, as shown in red?

![image](https://github.com/simple-icons/simple-icons/assets/1830867/fdc4c757-79d7-4904-8d8f-180be52fd413)